psp/src/lib.rs

```diff
@@ -12,7 +12,8 @@
     const_if_match,
     core_intrinsics,
     c_variadic,
-    lang_items
+    lang_items,
+    rustc_attrs,
```

Comment:
cargo fmt considers this comma a formatting error.

Also, could you update the cargo-psp version as well?